zebra-rpc/src/methods/get_block_template_rpcs.rs

```diff
@@ -1184,17 +1186,23 @@ where
             // Always zero for post-halving blocks
             let founders = Amount::zero();
 
+            let mut streams_total = Amount::zero();
+            let mut funding_streams_total = Amount::zero();
+            let mut lockbox_total = Amount::zero();
+
             let funding_streams =
                 funding_stream_values(height, &network).map_err(|error| Error {
                     code: ErrorCode::ServerError(0),
                     message: error.to_string(),
                     data: None,
                 })?;
+
             let mut funding_streams: Vec<_> = funding_streams
                 .iter()
-                .filter_map(|(receiver, value)| {
-                    let address = funding_stream_address(height, &network, *receiver)?;
-                    Some((*receiver, FundingStream::new(*receiver, *value, address)))
+                .map(|(receiver, value)| {
+                    streams_total = (streams_total + *value).expect("total is always valid");
```

Comment:
We could implement `Sum` for `Amount` to make this cleaner but not really needed to do now.

Reply:
We're getting the sum of the funding stream values in `miner_subsidy()`, could we do the same here?

```rust
    let total_funding_stream_amount: Result<Amount<NonNegative>, _> =
        funding_stream_values(height, network)?.values().sum();
```